package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "hx9-control-system",
+  "version": "1.0.0",
+  "description": "The control system consists of the pod operation backend and control station client. See the respective README files for setup instructions.",
+  "main": "index.js",
+  "dependencies": {
+    "husky": "^9.0.11"
+  },
+  "devDependencies": {
+    "pre-commit": "^1.2.2"
+  },
+  "scripts": {
+    "tsc": "cd control-station && tsc --noEmit && exit 0",
+    "eslint": "cd control-station && npx eslint . && exit 0",
+    "prettier": "cd control-station && prettier --write . && exit 0",
+    "cargoclippy": "cd pod-operation && cargo clippy && exit 0",
+    "cargocheck": "cd pod-operation && cargo check && exit 0",
+    "cargofmt": "cd pod-operation && cargo fmt && exit 0"
+  },
+  "pre-commit": ["prettier", "cargofmt"],
```

Comment:
Question: do we need to add the other scripts to this array?

Reply:
Yes, I will be adding them.